README.md

```diff
@@ -42,7 +42,7 @@ Installing
 
         $ pip install -r src/requirements.txt
 
-2. Modify ankisyncd.conf according to your needs
+2. Modify `/anki-sync-server/src/ankisyncd.conf` according to your needs, all subsequent commands refer to `/anki-sync-server/src/`.
```

Comment:
Please use a link i.e. [ankisyncd.conf](/src/ankisyncd.conf) instead of including path as text.

Reply:
updated accordingly, thx for the feedback!